book/07-git-tools/sections/searching.asc

```diff
@@ -111,8 +111,9 @@ If you need to be more specific, you can provide a regular expression to search
 ===== Line Log Search
 
 Another fairly advanced log search that is insanely useful is the line history search.
-This is a fairly recent addition and not very well known, but it can be really helpful.
-It is called with the `-L` option to `git log` and will show you the history of a function or line of code in your codebase.
+This can be really helpful.
+It is called with the `-L` option to `git log`.
+It will show you the history of a function or line of code in your codebase.
```

Comment:
The structure here is kind of stilted. Instead of these three sentences, what about just one:

> Simply run `git log` with the `-L` option, and it will show you the history of a function or line of code in your codebase.